src/scenes/PortalScene.ts

```diff
@@ -93,4 +92,7 @@ export class PortalScene implements SceneMapper {
   getModel() {
     return this.group;
   }
+  update() {
```

Comment:
If you make update a member function, outside calls to it won't have access to `this`. Instead, we should standardize the update function like so:
```javascript
update = () => {
  ...logic
}
```
#18 enforces this pattern